examples/java/rest/api/v2010/AwsCreator.java

```diff
@@ -128,7 +128,7 @@ public AwsCreator setTestObjectArray(final List<Object> testObjectArray){
         this.testObjectArray = testObjectArray;
         return this;
     }
-    public AwsCreator setTestAnyType(final Object testAnyType){
+    public AwsCreator setTestAnyType(final Map<String, Object> testAnyType){
```

Comment:
Need a unit test that verifies a map can be passed in here.

Reply:
@childish-sambino  Added a test for passing the map.